src/assets/scss/_give.scss

```diff
@@ -537,45 +561,51 @@ hr.horizontal-divider {
 
 
 .modal-meta{
-  color: lighten($colorGrayscale-text, 40%);
-  margin-bottom: $gutter*.25;
   display: block;
+  margin-bottom: $gutter * .25;
+  color: lighten($colorGrayscale-text, 40%);
   line-height: 1.15em;
 }
 
 .repeating-row {
 	clear: both;
 	line-height: 1em;
```

Comment:
Is there a configuration we could add to make indentation uniform? This jumps from two spaces to eight.

Reply:
Done